config/checkstyle-checks.xml

```diff
@@ -115,6 +115,13 @@
     <property name="matchAcrossLines" value="true" />
     <property name="message" value="Empty file is not allowed" />
   </module>
+  <module name="RegexpMultiline">
```

Comment:
I tested it and it helped me to find a couple that I missed.

![Screenshot from 2023-06-01 21-01-04](https://github.com/checkstyle/checkstyle/assets/23459549/9f7c57b1-300b-4677-a9ed-35bc05068671)